src/examples/src/form-bindings/App/template.html

```diff
@@ -16,7 +16,8 @@ <h2>Multi Checkbox</h2>
 <label for="john">John</label>
 <input type="checkbox" id="mike" value="Mike" v-model="checkedNames">
 <label for="mike">Mike</label>
-<p>Checked names: <pre>{{ checkedNames }}</pre></p>
```

Comment:
maybe just use `<span>` like other examples below this? 👀

Reply:
You're right, putting this in one line with `<span>` is better. 😆